src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -622,13 +633,25 @@ def get_image_uri(region_name, repo_name, repo_version=1):
         if supported_version:
             return get_xgboost_image_uri(region_name, supported_version[0])
 
-        logging.warning(
-            "There is a more up to date SageMaker XGBoost image. "
-            "To use the newer image, please set 'repo_version'="
-            "'%s'. For example:\n"
-            "\tget_image_uri(region, 'xgboost', '%s').",
-            XGBOOST_VERSION_1,
-            XGBOOST_VERSION_1,
-        )
     repo = "{}:{}".format(repo_name, repo_version)
     return "{}/{}".format(registry(region_name, repo_name), repo)
+
+
+def _xgboost_version_compare(repo_version):
+    """Compare xgboost image version with latest version
+
+    Args:
+        repo_version:
+    """
+    if repo_version == 1:
```

Comment:
`"latest"` is another tag that points to the legacy XGBoost image which we should check for.

Reply:
Added the "latest" check